### PR TITLE
Build scripts refactoring and Linux arm64

### PR DESCRIPTION
**Pull request by VladRassokhin**
Note that jars for aarch64 required, don't know where to grab them

### CODE REVIEW

File: jb/tools/linux/build.sh [hunk 1]
```diff
@@ -1,40 +1,48 @@
...
+*) print_usage 1 ;;
+esac
+
+script_path="$(readlink -f "$0")"
+script_dir="$(dirname "$script_path")"
+
+PATH="$script_dir:$PATH"
+
+log "*** BUILD NATIVE ***"
+bash "$script_dir/build_native.sh" $CLEAN
```
Comment (forantar) [inline jb/tools/linux/build.sh:42]: Why you removed `|| exit 1`?

Comment (forantar) [reply jb/tools/linux/build.sh:42]: I see - because of `pipefail`, ok good.

**Review by forantar — APPROVED** [review]

**Comment by forantar** [comment]
Thank you!

**Comment by forantar** [comment]
Pushed, details in: [JBR-4368](https://youtrack.jetbrains.com/issue/JBR-4368) Set up JCEF builds for Linux aarch64/arm64

**Comment by VladRassokhin** [comment]
Ah, I've created commits history with meaningful messages on purpose and now it's lost.

**Pull request closed without merging**